src/definitions/constants.rs

```diff
@@ -113,5 +97,19 @@ lazy_static! {
     pub static ref VALIDATE_ENTRY_POINT_SELECTOR: Felt252 =
         felt_str!("626969833899987279399947180575486623810258720106406659648356883742278317941");
 
+    pub static ref VALIDATE_RETDATA: Felt252 =
+        felt_str!("370462705988");
+
     pub static ref BLOCK_HASH_CONTRACT_ADDRESS: Address = Address(1.into());
 }
+
+// Indentation for transactions meant to query and not addressed to the OS.
```

Comment:
I don't understand this comment, can you add more description to it?

Reply:
Its the same comment in the original code (cairo-lang), I do not posses further information nor understanding

Reply:
Can you link to the original line? I have some suspicions about what it means but need to confirm.

Reply:
Here https://github.com/starkware-libs/cairo-lang/blob/10d257de5f49543757599cad9afaa5b4dd6d752b/src/starkware/starknet/definitions/constants.py#L65

Reply:
Oh, no dice. I thought the mention to indentation was to use scoping rules in Python to hide constants used only to determine the other constants, but it doesn't look like that.

Reply:
I think we should remove the comment if we don't understand it. Maybe put a link to the reference instead.